dist/package.json

```diff
@@ -55,7 +55,7 @@
     "validator": "^9.2.0",
     "validator.js": "^2.0.3",
     "validator.js-asserts": "^3.0.1",
-    "yargs": "^10.0.3"
+    "yargs": "^13.3.0"
```

Comment:
@rplopes the `dist/package.json` file does not need to be updated. Can you remove this change please?

Reply:
Done.